src/css/sdk.scss

```diff
@@ -1,4 +1,20 @@
+/*
+ * @forgerock/javascript-sdk-ui
+ *
+ * sdk.scss
+ *
+ * Copyright (c) 2020-2021 ForgeRock. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+ /**
+ * Import Twitter Bootstrap and its overrides from `ui-design.scss`
+ */
 @import 'ui-design';
```

Comment:
This is a reference to a file named "ui-design" and _not_ the library that was removed. To reduce risk, I'm trying to keep as much of the code and file names the same as before.